.github/workflows/wasmcloud-chart.yml

```diff
@@ -73,11 +73,15 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Lowercase the organization name for ghcr.io
+        run: |
+          echo "GHCR_REPO_NAMESPACE=${GITHUB_REPOSITORY_OWNER,,}" >>${GITHUB_ENV}
```

Comment:
Aren't we missing the package name here? For context, that's what we do for OCI https://github.com/wasmCloud/wasmCloud/blob/e383accd311b14e7bf85c536f37ae09e90a1d2c1/.github/workflows/wasmcloud.yml#L242

Reply:
Helm will automatically append the chart name to the remote it's given, so it'll turn `oci://ghcr.io/wasmcloud` into `oci://ghcr.io/wasmcloud/<chart-name>` (in our case ``oci://ghcr.io/wasmcloud/wasmcloud-chart`)

Reply:
Oh, I see, good to know, thanks!